gordo_components/builder/build_model.py

```diff
@@ -72,7 +72,22 @@ def build_model(
     # Cross validate
     logger.debug(f"Starting to do cross validation")
     start = time.time()
-    cv_scores = cross_val_score(model, X, y, cv=TimeSeriesSplit(n_splits=3))
+
+    scores: Dict[str, Any]
+    if hasattr(model, "score"):
+        cv_scores = cross_val_score(model, X, y, cv=TimeSeriesSplit(n_splits=3))
+        scores = {
+            "explained-variance": {
+                "mean": cv_scores.mean(),
+                "std": cv_scores.std(),
+                "max": cv_scores.max(),
+                "min": cv_scores.min(),
+                "raw-scores": cv_scores.tolist(),
+            }
+        }
+    else:
+        scores = {"message": "Unable to obtain scores for this model"}
```

Comment:
I think i prefer scores to just be an empty dict (or not exists at all). As a consumer I think maybe I would like to loop over all scores and e.g. print them (as we actually do ourself when `--print-vc-scores` is set`, and this means I need to look for the magic key "message" and try to interpret that to know if we have scores or not. If you want this message then maybe there is some other place we can put it? Or maybe just log it?

Reply:
Oh ya, I agree, that's much better.